docs/design/owt_with_webrtc_apis.md

```diff
@@ -1,19 +1,71 @@
 - Replace a track in the middle of a call.
 - Set custom encoding parameters, perhaps for simulcast.
 - Set preferred codecs.
 - Disable or enable RTX / RED / FEC.
-#### API Changes
+## API Changes
 - A new member `rtpTransceivers` will be added to `TransportSettings`. It returns an array `RTCRtpReceiver`s for RTP transport.
 - A new member `transport` will be added to `Publication` and `Subscription`. Developers could get `RTPTransceiver`s from its `rtpTransceivers` property.
-- A new method `addTransceiver(DOMString trackKind, sequence<RTCRtpEncodingParameters> sendEncodings)` will be added to `ConferenceClient`. It invokes `RTCPeerConnection.addTransceiver(trackKind, {direction:inactive, sendEncodings:sendEncodings})`, returns an `RTCRtpTransceiver`. Please note that direction is `inactive` until a `publish` with return transceiver is called.
-- The second parameter of `ConferenceClient.publish` accepts an `RTCRtpTransceiver` created by `RTCPeerConnection.addTransceiver`. When this method is called, certain `RTCRtpTransceiver`'s direction is changed to `sendonly`, and its sender's `setStreams` is called with the first parameter's `mediaStream`.
-#### Server Requirements
+- A new member `peerConnection` will be added to `ConferenceClient` for developers to get the `PeerConnection`.
+- The second parameter of `ConferenceClient.publish` accepts an `RTCRtpTransceiver` created by `RTCPeerConnection.addTransceiver`. `RTCRtpTransceiver`s' direction must be `sendonly`, and its sender's `setStreams` is called with the first parameter's `mediaStream`.
+## Server Requirements
 - `addTransceiver` and new `publish` needs renegotiation support.
-#### Remarks
-- Every transceiver could be associated with at most one `Publication` or one `Subscription`.
\ No newline at end of file
+## Remarks
+- Every transceiver could be associated with at most one `Publication` or one `Subscription`.
+## Examples
+### Replace a video track
+```
+const transceivers = publication.transport.rtpTransceivers;
+for (const transceiver of transceivers) {
+  if (transceiver.sender.track?.kind === 'video'){
+    transceiver.sender.replaceTrack(newTrack);
+  }
+}
+```
+
+### Publish a stream with codec preferences
+```
+const codecCapabilities = RTCRtpSender.getCapabilities('video').codecs;
+// Reorder codecCapabilities or remove some items.
+const transceivers = [];
+for (const track of mediaStream.getTracks()) {
+  const transceiver =
+      conference.peerConnection.addTransceiver(track, {
+        direction: 'sendonly',
+        streams: [stream],
+      });
+  if (track.kind==='video'){
+      transceiver.setCodecPreferences(codecCapabilities);
+  }
+  transceivers.push(transceiver);
+}
+const publication = await conference.publish(localStream, transceivers));
+```
+
+
+### Publish with a codec supports SVC.
+This example uses some WebRTC APIs may not be supported by all browsers.
+```
+const transceiver =
+    conference.peerConnection.addTransceiver(track, {
```

Comment:
This sample is a simulcast not SVC :)

Reply:
It's actually an example of spatial simulcast and temporal scalability. The configuration is the same as [example 7.1 of webrtc-svc](https://www.w3.org/TR/webrtc-svc/#simulcasttemporal-example*).